htdocs/projet/tasks/list.php

```diff
@@ -66,12 +66,10 @@
 $mine = $_REQUEST['mode'] == 'mine' ? 1 : 0;
 if ($mine) { $search_task_user = $user->id; $mine = 0; }
 
-$search_sday	= GETPOST('search_sday', 'int');
-$search_smonth	= GETPOST('search_smonth', 'int');
-$search_syear	= GETPOST('search_syear', 'int');
-$search_eday	= GETPOST('search_eday', 'int');
-$search_emonth	= GETPOST('search_emonth', 'int');
-$search_eyear	= GETPOST('search_eyear', 'int');
+$search_dateo_start = dol_mktime(0, 0, 0, GETPOST('search_dateo_startmonth', 'int'), GETPOST('search_dateo_startday', 'int'), GETPOST('search_dateo_startyear', 'int'));
```

Comment:
Because the value is coming from input user, the dol_mktime must contains the last parameter "tzuserrel"